packages/adapter-begin/index.js

```diff
@@ -33,6 +33,10 @@ export default function () {
 			const static_directory = resolve(static_mount_point);
 			const server_directory = resolve(join('src', 'shared'));
 
+			utils.update_ignores({
```

Comment:
can you not update `adapter-begin`? it's being fixed up and hosted in an external repo (currently https://github.com/ryanbethel/sveltekit-begin-adapter), so this change will conflict with the PR to point at the new home